### PR TITLE
Create DFS.py

### PR DESCRIPTION
**Pull request by avnishsingh516**

### CODE REVIEW

**Comment by ekdnam** [comment]
Good job! Add the functionality of taking the input of the graph from the user, rather than statically, and I will merge it!

**Comment by avnishsingh516** [comment]
please review the code.
I made the changes

File: Graph Theory/DFS/DFS.py [hunk 1]
```diff
@@ -46,12 +46,12 @@ def DFS(self, v):
...
-g.addEdge(3, 3) 
+E = int(input("Enter the number of edges:"))
+
+for i in range(0,E):
+  x, y = input("Enter a edges value: ").split()
+  g.addEdge(x, y) 
+
   
 print("Following is DFS from (starting from vertex 2)") 
 g.DFS(2)
```
Comment (ekdnam) [inline Graph Theory/DFS/DFS.py]: Can the starting vertex be dynamic?

**Review by ekdnam — CHANGES_REQUESTED** [review]

File: Graph Theory/DFS/DFS.py (hunk 1 above)
Comment (ekdnam) [reply Graph Theory/DFS/DFS.py]: I cannot find the new commit, so have mentioned the old commit here. (Thus we can see outdated)

Comment (avnishsingh516) [reply Graph Theory/DFS/DFS.py]: i made it dynamic.
please, review it 4358ddaf4ac41a10cf13d35070e43f3daed6821c

Comment (avnishsingh516) [reply Graph Theory/DFS/DFS.py]: 4358ddaf4ac41a10cf13d35070e43f3daed6821c

**Review by ekdnam — APPROVED** [review]

**Pull request merged**